Remotion/Web/Core/Res/Themes/NovaGray/HTML/StyleUtility.js

```diff
@@ -27,23 +27,23 @@ StyleUtility.CreateBorderSpans = function (selector)
 StyleUtility.AddBrowserSwitch = function ()
 {
```

Comment:
The ticket mentioned an API that can be used to do the detection, but it had the wrong name. We found the API again and we should use it first if available. If not we fall back to the user agent detection.

The API is https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgentData and it works in Chrome, Edge, and Opera. We need to add logic for the Browser detection and the Platform detection (below).